src/MonsterConfig.cpp

```diff
@@ -92,13 +96,15 @@ void cMonsterConfig::AssignAttributes(cMonster * a_Monster, const AString & a_Na
 	{
 		if (itr->m_Name.compare(a_Name) == 0)
 		{
-			a_Monster->SetAttackDamage   (itr->m_AttackDamage);
-			a_Monster->SetAttackRange    (itr->m_AttackRange);
-			a_Monster->SetSightDistance  (itr->m_SightDistance);
-			a_Monster->SetAttackRate     (static_cast<float>(itr->m_AttackRate));
-			a_Monster->SetMaxHealth      (static_cast<float>(itr->m_MaxHealth));
-			a_Monster->SetIsFireproof    (itr->m_IsFireproof);
-			a_Monster->SetBurnsInDaylight(itr->m_BurnsInDaylight);
+			a_Monster->SetAttackDamage      (itr->m_AttackDamage);
+			a_Monster->SetAttackRange       (itr->m_AttackRange);
+			a_Monster->SetSightDistance     (itr->m_SightDistance);
+			a_Monster->SetAttackRate        (static_cast<float>(itr->m_AttackRate));
+			a_Monster->SetMaxHealth         (static_cast<float>(itr->m_MaxHealth));
+			a_Monster->SetRelativeWalkSpeed (static_cast<float>(itr->m_RelativeWalkSpeed));
+			a_Monster->SetRelativeRunSpeed  (static_cast<float>(itr->m_RelativeRunSpeed));
```

Comment:
These functions take `double`, not `float`:
```suggestion
			a_Monster->SetRelativeWalkSpeed (itr->m_RelativeWalkSpeed);
			a_Monster->SetRelativeRunSpeed  (itr->m_RelativeRunSpeed);
```